app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,16 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+ 
+  def perform(initiator, benchmark_type, benchmark_result_type, category)
+    commit_objects = BenchmarkRun.where( initiator: initiator, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).limit(10)
```

Comment:
I think we can hardcode it to a specific number but this largely depends on what baseline we are considering to detect performance regression in this new commit.